__tests__/ncgamesapp.test.js

```diff
@@ -318,3 +318,23 @@ describe("POST /api/reviews/:review_id/comments", () => {
       });
   });
 });
+describe("GET:/api/reviews accepts queries", () => {
+  test("GET /api/reviews?order=asc responds with reviews sorted by date in ascending order", () => {
+    return request(app)
+      .get("/api/reviews?order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.reviews).toBeSortedBy("created_at", {
+          ascending: true,
+        });
+      });
+  });
+  test("GET /api/reviews?category=evilcategory throws 400 if entered category not in whitelist", () => {
+    return request(app)
+      .get("/api/reviews?category=evilcategory")
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("please enter valid category");
+      });
+  });
```

Comment:
Quite a few tests missing in this set:
- Can you filter by a valid category?
- Can you sort by different columns e.g. votes, created_at etc.
- What happens when you try to sort by a invalid column e.g. sort_by=bananas
- what happens when you try to order by a invalid order e.g. order=notValid
- When we don't send a query, do we get the default ordering of DESC by created_at?
- What happens when you try to query a valid category (exists in the database) that just doesn't have any reviews associated with (200 OK, empty arr response)

Also a category that isn't in the database should give a 404 not found instead of 400